packages/caliper-fabric/lib/adaptor-versions/v2/fabric-gateway-v2.js

```diff
@@ -489,6 +489,7 @@ class Fabric extends BlockchainInterface {
             invokeStatus.result = result;
             invokeStatus.verified = true;
             invokeStatus.SetStatusSuccess();
+            invokeStatus.SetID(transaction.getTransactionId());
```

Comment:
The TX ID should be set also for failed TXs.

Reply:
👍 Change made